### DISCUSSION TITLE
Instant rejection of student id

### DISCUSSION THREAD

**Discussion by jonahbkaplan** [community/community#52276] [GitHub Education]
Hi, I go to a secondary school in the UK, I've taken a photo of my school id which has my name, the school name and says "Valid until 31st July 2023" however this is at 90 degrees to the rest of the text. I get instantly rejected when i send this and it only says "we were unable to verify your academic affiliation" as I've believe I've fixed my previous rejection reasons.
Any help would be appreciated!

**Comment by samus-aran** [comment]
Hi @GingerNinja8896 👋 

Thank you for bringing this to our attention. We've passed this along to our internal team to investigate and when we have any updates we'll update this thread. 👍

**Reply by samus-aran** [reply]
Hi @GingerNinja8896 

We have updated your selected school and processed your application 👍 Let us know when you receive the GitHub verification email 🎉
